pkg/eventfilter/benchmarks/common_benchmark_test.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package benchmarks
+
+import (
+	"context"
+	"testing"
+
+	cloudevents "github.com/cloudevents/sdk-go/v2"
+
+	"knative.dev/eventing/pkg/eventfilter"
+)
+
+type FilterBenchmark struct {
+	name  string
+	arg   interface{}
+	event cloudevents.Event
+}
+
+// Avoid DCE
+var F eventfilter.Filter
+var R eventfilter.FilterResult
```

Comment:
IMO a meaningful name can be helpful here.

Reply:
Also, does it really need to be exported or even global variable?

Reply:
Yes, that's required to exclude the DCE

Reply:
How would you name it?

Reply:
https://dave.cheney.net/high-performance-go-workshop/dotgo-paris.html#fixing_the_benchmark

Reply:
Thanks for the pointer. Will `filter` and `filterResult` work?